.github/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+### Fixes: #[Add issue number here]
```

Comment:
Not all pull_requests fix an issue, some PRs may create a new page or make a new release. Maybe we should not include `Fixes` in header. And we can have a section like :
```
## Related issues
Closes #
```